tests/std/tests/VSO_0157762_feature_test_macros/test.cpp

```diff
@@ -317,6 +317,22 @@ STATIC_ASSERT(__cpp_if_constexpr == 201606L);
 #endif
 #endif
 
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, VS 2019 16.7p1
+#if _HAS_CXX20 && !defined(__EDG__) // TRANSITION, EDG

```

Comment:
I observe that this logic isn't really consistent with our other feature-test macro tests (it does nothing for C1XX instead of expecting the macro to not be defined), but I assume it's intentional, and it's TRANSITION commented, so no change requested.

Reply:
Yes, this is deliberate to keep the test working with both internal MSVC that supports the feature and external MSVC that does not. (We haven't bumped `_MSC_VER` yet.)